src/EnergyPlus/ChillerAbsorption.cc

```diff
@@ -255,20 +237,20 @@ namespace ChillerAbsorption {
 
         DataIPShortCuts::cCurrentModuleObject = moduleObjectType;
 
-        numBlastAbsorbers = inputProcessor->getNumObjectsFound(DataIPShortCuts::cCurrentModuleObject);
+        chillers.numBlastAbsorbers = inputProcessor->getNumObjectsFound(DataIPShortCuts::cCurrentModuleObject);
 
-        if (numBlastAbsorbers <= 0) {
+        if (chillers.numBlastAbsorbers <= 0) {
             ShowSevereError("No " + DataIPShortCuts::cCurrentModuleObject + " equipment specified in input file");
             // See if load distribution manager has already gotten the input
             ErrorsFound = true;
         }
 
-        if (allocated(BLASTAbsorber)) return;
+        if (allocated(chillers.BLASTAbsorber)) return;
 
-        BLASTAbsorber.allocate(numBlastAbsorbers);
+        chillers.BLASTAbsorber.allocate(chillers.numBlastAbsorbers);
```

Comment:
Missed ```auto &``` here. Add it?

Reply:
No, the allocate call just returns a reference to the original object, no need to capture it (in our use cases, anyway).